### ISSUE TITLE
Text glitch in first fight screen

### ISSUE THREAD

**Issue by thingywhat** [sprpgs/PotionWars#78]
I noticed that during the intro of the first fight in the guild, it appears as though text is inserted twice at two different heights and overlaps itself:

![glich please](https://cloud.githubusercontent.com/assets/6616922/9320447/79ca5fc8-4517-11e5-9be5-d359bdb3262c.png)

It is present as late as commit c0c3f0be2057b9b88cae1d1adcb4d7a9df057372 running on Debian 7 with Python 2.7.3.

**Comment by sprpgs** [comment]
Huh. Probably a side-effect of the fact that I took a very dumbass "roll my own" approach to the GUI (I forgot toolkits were a thing), and things aren't refreshing correctly. That'll (hopefully) be fixed after episode 2, when I rip out all of my horrible GUI code and replace it with a proper toolkit.

But thanks for pointing this out!

**Comment by thingywhat** [comment]
Well... I sometimes like to reinvent the wheel myself, because there's no better way to learn something than to make it yourself.

But yeah, no prob..!

**Comment by sprpgs** [comment]
Indeed. If nothing else, it teaches you that the wheel is hard as hell to get right, and you're better off using someone else's.
